tests/test_yaml_helpers.py

```diff
@@ -44,7 +44,7 @@ def test_save_yaml(content, result):
     try:
         save_yaml(temp_file.name, content)
         yaml_content = open_yaml(Path(temp_file.name))
-        assert yaml_content == yaml.load(result)
+        assert yaml_content == yaml.safe_load(result)
```

Comment:
maybe we want to change this, now that we have changed the way of loading the YAML.

Reply:
Eh I think in the test it's ok no?